### PR TITLE
add README for llm-inference-cpu

### PR DESCRIPTION
**Pull request by Zhengjin-Wang**

### CODE REVIEW

File: docker/llm/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Use LLM on docker
+
+### Install docker
+
+The docker install package can be downloaded from [here](https://www.docker.com/get-started/)
+
+For Windows users, make sure Hyper-V is enabled on your computer.
```
Comment (glorysdj) [inline docker/llm/README.md:7]: add a reference link to https://docs.docker.com/desktop/install/windows-install/

Comment (Zhengjin-Wang) [reply docker/llm/README.md]: added

> add a reference link to https://docs.docker.com/desktop/install/windows-install/

**Comment by glorysdj** [comment]
Getting started with BigDL LLM on Windows
Install Docker
Pull bigdl-llm-cpu Docker image
Prepare LLM Models
Start bigdl-llm-cpu container
Getting started with chat
Getting started with tutorials

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**